ngraph_bridge/ngraph_rewrite_pass.cc

```diff
@@ -221,26 +221,26 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
     // to be attached to the nodes
     // Precedence Order: Env Variable > BackendManager
     std::unordered_map<std::string, std::string> config_map;
-    string backend_creation_string;
-    // GetCurrentlySetBackendName could return GPU:0 (not just GPU)
+    string backend_name;
     TF_RETURN_IF_ERROR(
-        BackendManager::GetCurrentlySetBackendName(&backend_creation_string));
+        BackendManager::GetCurrentlySetBackendName(&backend_name));
 
-    // splits into {"ngraph_backend", "ngraph_device_id"}
+    // splits into {"ngraph_backend", "_ngraph_device_config"}
     config_map = BackendManager::GetBackendAttributeValues(
-        backend_creation_string);  // SplitBackendConfig
+        backend_name);  // SplitBackendConfig
+    backend_name = config_map.at("ngraph_backend");
     config_map.erase("ngraph_backend");
 
     if ((std::getenv("NGRAPH_TF_LOG_0_DISABLED") == nullptr)) {
-      NGRAPH_VLOG(0) << "NGraph using backend: " << backend_creation_string;
+      NGRAPH_VLOG(0) << "NGraph using backend: " << backend_name;
```

Comment:
I think these are extraneous changes from updating/merging to master. I think we want the name to remain `backend_creation_string` instead of `backend_name` and `ngraph_device_id` instead of `_ngraph_device_config`

Reply:
I see 22 files changed in this PR. It seems there might be other such merge related changes. please recheck that we are not making any major changes to anything unrelated.